api/v1alpha1/terraform_types.go

```diff
@@ -120,6 +120,10 @@ type TerraformSpec struct {
 	// +required
 	Interval metav1.Duration `json:"interval"`
 
+	// Duration to retry a state lock.
+	// +optional
+	LockTimeout string `json:"lockTimeout,omitempty"`
+
```

Comment:
Should this be part of the `TFStateSpec`?

Reply:
Also this duration would be of the Duration type (from the metav1 package).

Reply:
thanks for the feedback, wasn't sure about these two 👌 changed